src/main.ts

```diff
@@ -115,6 +115,13 @@ class ChronerEngine extends Engine<never, ChronerTask> {
   }
 }
 
+class ChronerStrategy extends CombatStrategy {
+  constructor(macro: Macro) {
+    super();
```

Comment:
why is echidna in this repository phred

Reply:
i think this was the correct pattern but it also doesn't matter very much

Reply:
Reverted!